index.bs

```diff
@@ -568,13 +578,23 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. Let |issuedRequests| and |currentlyAvailableAuthenticators| be new [=ordered sets=].
 
-1. For each |authenticator| currently available on this platform, if
-    <code>|options|.{{MakeCredentialOptions/attachment}}</code> is [=present|not present=] or its value matches
-    |authenticator|'s [=attachment modality=], [=set/append=] |authenticator| to |currentlyAvailableAuthenticators|. 
-
 1. If |currentlyAvailableAuthenticators| [=list/is empty=], return a {{DOMException}} whose name is 
     "{{NotFoundError}}", and terminate this algorithm.
 
+1. For each [=authenticator=] currently available on this platform, process each option in 
+    |options|.{{MakeCredentialOptions/authenticatorSelection}} by checking if the [=authenticator=] satisfies the 
+    requirement of the option. 
+        1. Check if |attachment| is [=present|not present=] or its value matches
```

Comment:
Try to avoid "Check if *X*" in specifications. That sort of sentence tends to fail to say what to do if *X* is true or false. Just "If *X*, then *Y*. Otherwise, *Z*." is clearer.